shell/platform/android/test/io/flutter/embedding/engine/FlutterJNITest.java

```diff
@@ -43,5 +43,12 @@ public void onFlutterUiNoLongerDisplayed() {}
 
     // --- Verify Results ---
     assertEquals(1, callbackInvocationCount.get());
+
+    // --- Execute Test ---
+    flutterJNI.onDisplayPlatformView(0, 0, 0, 0, 0);
```

Comment:
hmm. I think this needs to go in a separate  `@Test`. Something like:

```java
@Test
public void onDisplayPlatformView() {
  FlutterJNI flutterJNI = new FlutterJNI();
  flutterJNI.setPlatformViewsController(new PlatformViewsController() {
      // Override onDisplayPlatformView and increase counter.
  });

  flutterJNI.onDisplayPlatformView(0, 0, 0, 0, 0);
  // assert that counter increased.
}
```

Reply:
Typically unit tests are testing a single "unit" of functionality. Here the "unit" is that `PlatformViewsController#onDisplayPlatformView` is called.

Reply:
Also, bonus point if we check that it's called with the right arguments. e.g. the id, x, y, width, height should match the ones you passed, so I would actually give some real values to the call:

```java
 flutterJNI.onDisplayPlatformView(/* viewId=*/ 1, /* x=*/ 10, /* y=*/ 20, /* width=*/ 100 , /* height=*/ 200);
```

Then, collect this values in the overridden method and assert that they are the same as the one you passed.

Reply:
You can use Mockito to achieve this. See: https://github.com/flutter/engine/blob/288b66b1b663ce5c52a6b5f2a97a4b7af5e8a6ad/shell/platform/android/test/io/flutter/embedding/engine/FlutterEngineTest.java#L110